mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.erb

```diff
@@ -699,7 +700,6 @@ resource "google_container_cluster" "cluster" {
 }
 `, clusterName, clusterName)
 }
-<% end -%>
```

Comment:
Should this line be deleted? The following log suggests that the `end` matching is not correct.

```
2024-07-03 05:20:27.234 JST
Step #11 - "tpgb-head": I, [2024-07-02T20:20:18.269515 #170] INFO -- : Compiling common files for terraform
2024-07-03 05:20:27.234 JST
Step #11 - "tpgb-head": #<Thread:0x00007f01c81a7590 /mm-terraform-beta-head/mmv1/provider/terraform.rb:189 run> terminated with exception (report_on_exception is true):
2024-07-03 05:20:27.234 JST
Step #11 - "tpgb-head": /usr/local/lib/ruby/3.1.0/erb.rb:905:in `eval': (erb):11083: syntax error, unexpected end-of-input, expecting `end' (SyntaxError)